tests/conftest.py

```diff
@@ -52,6 +52,39 @@
 ]
 
 
+@pytest.fixture(scope='module')
+def es(appctx):
+    """Setup and teardown all registered Elasticsearch indices.
+
+    Scope: module
+    This fixture will create all registered indexes in Elasticsearch and remove
+    once done. Fixtures that perform changes (e.g. index or remove documents),
+    should used the function-scoped :py:data:`es_clear` fixture to leave the
+    indexes clean for the following tests.
+    """
+    from elasticsearch.exceptions import RequestError
```

Comment:
please put it at the top of the file if it is possible

Reply:
done